### PR TITLE
KOGITO-1107: Ensure prod daily build pass kie-wb-common selenium test

### PR DESCRIPTION
**Pull request by jomarko**
Ensemble together with:
- https://github.com/kiegroup/kie-wb-common/pull/3217

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@manstis @mbiarnes @mareknovotny please re-review the new smaller ensemble as @mareknovotny asked me to eliminate changes in `kie-parent`. I hope this could work.

File: job-dsls/jobs/dailyBuild_pipeline.groovy [hunk 1]
```diff
@@ -345,15 +345,22 @@ matrixJob("${folderPath}/daily-build-${baseBranch}-jbpmTestContainerMatrix") {
...
+echo "KIE version $kieVersion - kie-wb-common"
+wget -q http://\\${LOCAL_NEXUS_IP}:8081/nexus/content/repositories/kieAllBuild-$baseBranch/org/kie/kie-wb-common/$kieVersion/kie-wb-common-$kieVersion-project-sources.tar.gz -O sources.tar.gz
+tar xzf sources.tar.gz
+rm sources.tar.gz
+mv kie-wb-common-$kieVersion/* .
+rmdir kie-wb-common-$kieVersion'''
 
 matrixJob("${folderPath}/daily-build-${baseBranch}-kieWbTestsMatrix") {
-    description("This job: <br> - Runs the KIE Server integration tests on mutiple supported containers and JDKs <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
+    description("This job: <br> - Runs the KIE wb integration tests on mutiple supported containers and JDKs <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
```
Comment (mareknovotny) [inline job-dsls/jobs/dailyBuild_pipeline.groovy]: just an optional thing, do we use uppercase shortcut here? s/wb/WB/

Comment (jomarko) [reply job-dsls/jobs/dailyBuild_pipeline.groovy]: Applied

**Comment by jomarko** [comment]
@kiegroup/gatekeepers could we please merge?

**Pull request merged**